.github/workflows/ci.yml

```
@@ -1,24 +1,39 @@
-name: Build Docker images
+name: Build and Push Docker Images (v4)
+
+env:
+  DOCKERHUB_USERNAME: molluxmollux
 
 on:
-  pull_request:
+  push:
 
 jobs:
   docker:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         image: [apache, fpm]
+
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Log in to Dockerhub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ env.  DOCKERHUB_USERNAME }}
```

Is the space here intentional?
```suggestion
          username: ${{ env.DOCKERHUB_USERNAME }}
```